modules/developer_manual/pages/app/advanced/l10n.adoc

```diff
@@ -188,3 +187,29 @@ myapp/l10n
 
 You then just need the .php, .json and .js files for a working localized
 app.
+
+
+[[how-to-automatically-sync-translations]]
+How to automatically sync translations
```

Comment:
Please [title case](http://www.grammar-monster.com/lessons/capital_letters_title_case.htm) this header and use [AsciiDoc header formatting](https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#section-titles), instead of the reStructuredText style.

Reply:
Could you not just tell me how to write it instead of sending me to the doc page?

Reply:
All the other titles in that document are in the same format, so I just copy pasted it.